src/app/tests/suites/certification/Test_TC_FAN_2_4.yaml

```
@@ -47,3 +49,5 @@ tests:
       response:
           constraints:
               type: bitmap8
+              minValue: 0
+              maxValue: 2
```

It is possible that a fan could support all bits simultaneously.
```suggestion
              maxValue: 3
```